apps/studio/src/features/editing-experience/components/RootStateDrawer.tsx

```diff
@@ -134,168 +215,272 @@ export default function RootStateDrawer() {
     savedPageState.content[0]?.type === "hero"
 
   const hasChildpagesBlock = pageLayout === ISOMER_USABLE_PAGE_LAYOUTS.Index
+  const isCustomIndexPage =
+    type === ResourceType.IndexPage && pageLayout !== "index"
```

Comment:
nit: this should be `isCustomContentIndexPage` or sometihng similar because we can have index pages which are custom (ie, have content) and have `layout: index`

Reply:
Updated!